Marlin/src/feature/tmc_util.h

```diff
@@ -53,10 +53,8 @@
 #define CHOPPER_PRUSAMK3_24V { 4,  1, 4 }
 #define CHOPPER_MARLIN_119   { 5,  2, 3 }
 
-#if ENABLED(MONITOR_DRIVER_STATUS)
-  #ifndef MONITOR_DRIVER_STATUS_INTERVAL_MS
-    #define MONITOR_DRIVER_STATUS_INTERVAL_MS 500u
-  #endif
+#if ENABLED(MONITOR_DRIVER_STATUS) && !defined(MONITOR_DRIVER_STATUS_INTERVAL_MS)
+  #define MONITOR_DRIVER_STATUS_INTERVAL_MS 500u
```

Comment:
As a style decision, could we favor `constexpr` for default values like this where the value isn't exposed to the user?

Reply:
The pre-compiler can't look for the existence of a `constexpr` variable. I assume that `MONITOR_DRIVER_STATUS_INTERVAL_MS` is meant to be defined one of the config files when needed, in which case it should still be a `#define`. There's no added value in also defining a `constexpr` to hold the value, though one might mirror it in a `static` class variable.

Reply:
I completely missed the check there.
I don't actually think it needs to be configurable but if you want to keep as a sort of "hidden option" then whatever works.
It would make sense if the defaults or "magic numbers" were placed under TMCStorage class.

Reply:
If there are defaults that Marlin would need to work with directly, you could provide those default values, either as defines or as `static inline` methods. Or, you could just provide methods to set each register to its default. Or, a combination….